core/common/src/main/java/alluxio/AlluxioConfiguration.java

```diff
@@ -153,40 +177,12 @@
    */
   Map<String, String> getNestedProperties(PropertyKey prefixKey);
 
-  /**
-   * @return a map of the properties represented by this configuration,
-   *         including all default properties
-   *
-   * @param options options for getting configuration value
-   */
-  Map<String, String> toMap(ConfigurationValueOptions options);
```

Comment:
do we want to support `toMap` for instanced conf in the future? If yes, we may want to make this method stay in the interface?

Reply:
done